includes/check-local-changes.yml

```diff
@@ -0,0 +1,9 @@
+# requires skip_local_rpm_check (bool, dafault to false) to be set
+---
+  - name: Verify local changes
+    script: check_monkey_patches.sh
```

Comment:
You can just do `rpm -V --noconfig obs-api` instead...

Reply:
Didn't know that, I'll try it later